components/popover/demo/placement.md

```diff
@@ -27,7 +27,7 @@ const content = (
 const buttonWidth = 70;
 
 ReactDOM.render(
-  <div className="demo">
+  <div className="demo" style={{ direction: 'ltr' }}>
```

Comment:
Do not change origin demo.

Reply:
If i remove this style from demo, it will not displays correctly in rtl direction.

![antd-dir-rtl-in-demo2](https://user-images.githubusercontent.com/12451743/71621397-52ce2180-2be4-11ea-9570-2bdd178ce768.gif)

I can handle it by css with demo id like this:
``` less
.ant-row-rtl #components-tooltip-demo-placement {
    direction: ltr;
}
```
Which way do you prefer?

Reply:
Prefer additional style instead of inline style : )